scripts/upload_zoom_recordings.py

```diff
@@ -126,35 +118,21 @@ def get_youtube_client():
             print(f'  Download from {url}...')
             filepath = download_file(url, tmpdirname)
             title = f'{meeting["topic"]} - {pretty_date(meeting["start_time"])}'
+
             # These characters don't work within Python subprocess commands
             chars_to_strip = '<>'
             title = re.sub('['+chars_to_strip+']', '', title)
-            command = [
-                "youtube-upload", filepath,
-                "--title=" + title,
-                "--playlist=" + DEFAULT_YOUTUBE_PLAYLIST,
-                "--category=" + DEFAULT_YOUTUBE_CATEGORY,
-                "--license=" + DEFAULT_VIDEO_LICENSE,
-                "--recording-date=" + fix_date(meeting['start_time']),
-                "--privacy=unlisted",
-                "--client-secrets=client_secret.json",
-                "--credentials-file=.youtube-upload-credentials.json"
-            ]
+
             print('  Adding to main playlist: Uploads from Zoom')
+            video_id = initialize_upload(youtube,
+                              filepath,
+                              title=title,
+                              category=28, # TODO: get category by name: DEFAULT_YOUTUBE_CATEGORY,
+                              license=DEFAULT_VIDEO_LICENSE,
+                              recording_date=fix_date(meeting['start_time']),
+                              privacy_status='unlisted')
             
-            try:
-                video_id = check_output(command, stderr=PIPE).strip().decode('utf-8')
-            except CalledProcessError as error:
-                print(f'  Upload failed with message:\n'
-                      f'{error.stderr.decode("utf-8")}'
-                      f'{error.stdout.decode("utf-8")}',
-                      file=sys.stderr)
-                sys.exit(1)
-
-            # TODO: we could use this client to upload the video,
-            # which would save on API calls if we have > 1 video.
-            youtube = get_youtube_client()
-            playlist_name = None
+            playlist_name = DEFAULT_YOUTUBE_PLAYLIST
```

Comment:
This looks like the behavior has changed a bit to me. Before, we *always* added the video to `DEFAULT_YOUTUBE_PLAYLIST` and then *additionally* added it to a playlist based on meeting topic (if applicable).

Now we only add it to `DEFAULT_YOUTUBE_PLAYLIST` *or* the topic-based playlist, but not both.